opa/metadata.csv

```diff
@@ -0,0 +1,4 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+opa.request.duration.count,count,,,,The count of the HTTP request latencies in seconds for the OPA service,1,opa,opa.req_duration.cnt
```

Comment:
We can probably add a unit here

Reply:
what do you mean? to change the metric name to `opa.request.duration.seconds.count` ?

Reply:
I mean setting the `unit_name` field here like:
```suggestion
opa.request.duration.count,count,,second,,The count of the HTTP request latencies in seconds for the OPA service,1,opa,opa.req_duration.cnt
```